redis/src/test/scala/zio/redis/KeysSpec.scala

```diff
@@ -112,6 +134,27 @@ trait KeysSpec extends BaseSpec {
           restored <- get(key)
         } yield assert(restore)(isRight) && assert(restored)(isSome(equalTo(value)))
       },
+      suite("migrate")(
+        testM("migrate key to another redis server (copy and replace)") {
+          for {
+            key       <- uuid
+            value     <- uuid
+            _         <- set(key, value)
+            response  <- migrate("redis2",
+                                  6379,
+                                  key,
+                                  0L,
+                                  JavaDuration.ofMillis(5000),
+                                  copy = Option(Copy),
+                                  replace = Option(Replace),
+                                  keys = None)
+            value2    <- get(key).provide(secondRedisService)
+          } yield
```

Comment:
Trouble is this doesn't compile because secondRedisService is not of the right type for get
[error]  found   : zio.ZLayer[Any with Any,zio.redis.RedisError.IOError,zio.Has[RedisExecutorWrapper]]
[error]  required: zio.redis.RedisExecutor
[error]     (which expands to)  zio.Has[zio.redis.RedisExecutor.Service]
[error]             value2    <- get(key).provide(secondRedisService)
[error]                                           ^
[error] one error found